json_journal/data_analysis.py

```diff
@@ -1,45 +1,84 @@
+'''Functions for various analysis on data'''
+COLON = ':'
```

Comment:
This is a bad constant name. What are we using the constant for? Use the name as a chance to concisely explain the constant's intended use.

Reply:
done